packages/strapi-plugin-content-manager/admin/src/containers/EditView/DeleteLink/index.js

```diff
@@ -14,7 +14,15 @@ import { connect, select } from './utils';
 
 const getRequestUrl = path => `/${pluginId}/explorer/${path}`;
 
-const DeleteLink = ({ canDelete, clearData, dataId, isCreatingEntry, isSingleType, slug }) => {
+const DeleteLink = ({
+  canDelete,
+  clearData,
+  dataId,
+  isCreatingEntry,
+  isSingleType,
+  property,
```

Comment:
Can you rename this prop to something like `eventProperty` or `trackerProperty`.